spec/requests/organisations_controller_spec.rb

```diff
@@ -129,6 +129,7 @@
           schemes[2].update!(service_name: "baa", owning_organisation: user.organisation)
```

Comment:
the nittiest of nits: while we're here, line 126 has a typo, should be "alphabetical", probably worth doing a find and replace :)